services/ui-src/src/utils/reports/reports.ts

```diff
@@ -53,3 +53,14 @@ export const createReportId = (
   const reportId = [state, programNameWithDashes, dueDateString].join("_");
   return reportId;
 };
+
+export const sortReportsOldestToNewest = (
+  reportsArray: { createdAt: number }[]
+) => {
+  return reportsArray.sort(function (
+    stateA: { createdAt: number },
+    stateB: { createdAt: number }
+  ) {
+    return stateA.createdAt - stateB.createdAt;
+  });
+};
```

Comment:
Nit: Could actually get even fancier here by using arrow functions/implicit return and only referencing the createdAt key when needed:

```suggestion
export const sortReportsOldestToNewest = (
  reportsArray: { createdAt: number }[]
) => reportsArray.sort((stateA, stateB) => stateA.createdAt - stateB.createdAt);
```

Reply:
condensed!